website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-supported-dbt-versions.md

```diff
@@ -1,13 +1,29 @@
 ---
-title: "Supported dbt versions"
-id: "cloud-supported-dbt-versions"
+title: "Configuring dbt version"
```

Comment:
This title sits a little strangely for me. "Configuring dbt version" 🤔 🤔 🤔 

"Configuring the dbt versions"?
"Configuring the dbt version"?
"Setting the dbt version"?
idk!

Reply:
Totally agree on the name. Could not for the life of me figure out a good one to go with the night I wrote this.